Global frequency set at 1000000000000 ticks per second
warn: No dot file generated. Please install pydot to generate the dot file and pdf.
src/dev/net/tcp_iface.cc:97: info: tcp_iface listening on port 2201
src/dev/net/tcp_iface.cc:177: info: Link okay  (iface:0 -> (node:2103825632, iface:21932))
src/dev/net/tcp_iface.cc:274: panic: send() failed: Bad file descriptor
Memory Usage: 132932 KBytes
Program aborted at tick 0
--- BEGIN LIBC BACKTRACE ---
./build/ARM/gem5.fast(_ZN4gem515print_backtraceEv+0x19)[0x55ac7d28aef9]
./build/ARM/gem5.fast(_ZN4gem512abortHandlerEi+0x3d)[0x55ac7d2a5e9d]
/lib/x86_64-linux-gnu/libc.so.6(+0x3bfd0)[0x7f060096cfd0]
/lib/x86_64-linux-gnu/libc.so.6(+0x8ad3c)[0x7f06009bbd3c]
/lib/x86_64-linux-gnu/libc.so.6(gsignal+0x12)[0x7f060096cf32]
/lib/x86_64-linux-gnu/libc.so.6(abort+0xd3)[0x7f0600957472]
./build/ARM/gem5.fast(+0x18a870f)[0x55ac7b6fc70f]
./build/ARM/gem5.fast(_ZN4gem58TCPIface7sendTCPEiPKvj+0x524)[0x55ac7cb7ebe4]
./build/ARM/gem5.fast(_ZN4gem58TCPIface19establishConnectionEv+0x194)[0x55ac7cb7f664]
./build/ARM/gem5.fast(_ZN4gem59DistIface4initEPKNS_5EventEm+0x17)[0x55ac7cb75817]
./build/ARM/gem5.fast(+0x33e3305)[0x55ac7d237305]
./build/ARM/gem5.fast(+0x1a9ba48)[0x55ac7b8efa48]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(+0x1c9bc3)[0x7f0601522bc3]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(_PyObject_MakeTpCall+0x7d)[0x7f06014d4e8d]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(_PyEval_EvalFrameDefault+0x4a63)[0x7f0601462c63]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(+0x3ccd3c)[0x7f0601725d3c]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(PyEval_EvalCode+0xc1)[0x7f06015c1891]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(+0x268fb0)[0x7f06015c1fb0]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(+0x1ca0c2)[0x7f06015230c2]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(PyObject_Vectorcall+0x33)[0x7f06014d6633]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(_PyEval_EvalFrameDefault+0x4a63)[0x7f0601462c63]
/usr/lib/x86_64-linux-gnu/libpython3.11.so.1.0(+0x26ab4a)[0x7f06015c3b4a]
./build/ARM/gem5.fast(+0x33b87a8)[0x55ac7d20c7a8]
./build/ARM/gem5.fast(main+0x211)[0x55ac7d2a7d31]
/lib/x86_64-linux-gnu/libc.so.6(+0x271ca)[0x7f06009581ca]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x85)[0x7f0600958285]
./build/ARM/gem5.fast(_start+0x21)[0x55ac7b6ef201]
--- END LIBC BACKTRACE ---
For more info on how to address this issue, please visit https://www.gem5.org/documentation/general_docs/common-errors/ 

